### DISCUSSION TITLE
Possibilidade de integrar o SonarCloud

### DISCUSSION THREAD

**Discussion by andrewarosario** [techexpertspro/portal-ascepa#41] [Ideas]
O SonarCloud é uma plataforma de análise de qualidade de código e segurança fornecida como um serviço (SaaS). Ele ajuda desenvolvedores e equipes a melhorar a qualidade do código e a detectar vulnerabilidades em tempo real, integrando-se facilmente com vários serviços de hospedagem de código, como o GitHub.

É possível integrar o o SonarCloud com um repositório GitHub de forma gratuita. O SonarCloud oferece um plano gratuito para projetos open-source.

**Comment by FelipeDuarteLuna** [comment]
Bom dia Andrew,

Sou a favor da integração, visto que isso é processo quase que padrão no mundo corporativo e fora que eleva o nível de codificação do projeto, identifica bugs, possiveis vulnerabilities e segurança.  Aquele padrão selo de qualidade.

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/7773d2dd-95ca-4075-b1ce-076b3c85c154)

**Comment by FelipeDuarteLuna** [comment]
Demanda realizada com sucesso, SonarCloud foi integrado a Pipeline ci/cd do Portal - Ascepa. 

https://sonarcloud.io/organizations/techexpertspro/projects

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/79662044-da27-41df-a66e-9b2612044a6f)

![image](https://github.com/techexpertspro/portal-ascepa/assets/29357935/4c8d3d7d-1357-44dc-865f-caa77247b5da)
